modules/rosa-update-cli-tool.adoc

```diff
@@ -0,0 +1,17 @@
+
+// Module included in the following assemblies:
+//
+// * rosa-release-notes/rosa-release-notes.adoc
+:_content-type: REFERENCE
+[id="updating_rosa_cli{context}"]
+== Updating the ROSA CLI tool
+
+You can download the latest version of the ROSA CLI from the Hybrid Cloud Console.
+
+To update to the latest version of the ROSA CLI:
+
+. Download the file from the link:https://console.redhat.com/openshift/downloads[Hybrid Cloud Console].
+. Unzip the downloaded file.
+. In the terminal, type `sudo mv rosa /usr/bin/rosa` then press **Enter**.
+//. In the terminal, type `chmod a+x /usr/bin/rosa` to make the ROSA binary you downloaded executable.
+. Type `rosa version` in the terminal to confirm your version. "Your ROSA CLI is up to date." is displayed when you have updated.
```

Comment:
Same here as above. Include "Your ROSA CLI is up to date." as output from the command.